src/react-app-env.d.ts

```diff
@@ -1,2 +1,17 @@
 // eslint-disable-next-line
 /// <reference types="react-scripts" />
+
+interface Good {
+  id: number;
+  name: string;
+  color: string;
+}
+
+interface Prop {
+  list: Good[];
+}
```

Comment:
Некорректное название, уточни чьи именно пропсы
```suggestion
interface GoodsListProps {
  list: Good[];
}
```